README-dev.md

```diff
@@ -66,57 +91,46 @@ The wrappers are built with the plugin. `dvp build` has a hidden `--dev` flag. T
 vsdk_root = /path/to/vsdk_repo_root
 ```
 
-### Versioning
-The first thing to do is to change the version number. There are _two_ places where the version needs to be changed unfortunately. For almost all cases, this will simply involve incrementing the "build number", which is the three-digit number at the very end of the version.
-
-1. In the root of the SDK codebase, open the `build.gradle` file, and change the version.
-2. In `tools/src/main/python/dlpx/virtualization/_internal/settings.cfg` change the `package_version` property.
-
-The two versions should be identical.
-
-This repository is going through a transition in which Gradle will eventually be removed. The version string will eventually only live in the individual packages themselves.
-
-### Testing
-
-Currently, there are three types of SDK testing: unit, manual, and blackbox.
+## Testing
 
-#### Unit Testing
+Currently, there are three types of SDK testing: unit, manual, and functional (blackbox).
 
-Running `./gradlew test` from the top level of the repository will run all SDK unit tests. Smaller sets of tests can be run from inside each directory (`common`, `platform`, etc.) by going into that directory and running `../gradlew test`. These can also be run through your IDE.
+### Unit Testing
 
-#### Testing sdk-gate changes with app-gate code
+Go into one of the package directories (common, dvp, libs, platform, tools) and run the following commands (if you haven't done it already):
+1. Install the package's development dependencies: `pip install -r requirements.txt`.
+2. Install the package itself in editable mode: `pip install -e .`.
+3. Run unit tests: `python -m pytest src/main/python`.
```

Comment:
Just adding this comment so we can figure out if people can actually just run `pytest`